R/helpers.R

```diff
@@ -26,6 +26,54 @@ groups_by_filter <- function(covariate, nbins, ties.method="random", seed=NULL){
 	as.factor(ceiling( rfs* nbins))
 }
 
+#' Stratify hypotheses based on increasing value of the covariate 
+#'
+#'  Hypotheses are stratified into nbins different strata of (approximately) equal size based on
+#' increasing value of the covariate
+#' extension of `groups_by_filter` for multidimensional covariates, per-covariate quantiles
+#'
+#' @param covariates Numeric vector of ordinal covariates based on which the stratification will be done.
+#' @param nbins Integer, number of groups/strata into which p-values will be split based on covariate.
+#' @param ties.method Character specifying how ties are treated, see \code{\link{rank}} function.
+#' @param seed Integer, specifies random seed to be used when ties.method=="random".
```

Comment:
Wrap such code examples in `\code{ties.method=="random"}` so that they are rendered nicely.

Reply:
Thank you, I adressed this and also updated some documantation in `ihw_convex.R` accordingly